bundler/lib/bundler/cli.rb

```diff
@@ -290,6 +290,8 @@ def install
       "Prefer updating only to next minor version"
     method_option "major", :type => :boolean, :banner =>
       "Prefer updating to next major version (default)"
+    method_option "pre", :type => :boolean, :banner =>
```

Comment:
definitely nice to have parity with `gem list "\A$GEM_NAME\z" --all --prerelease --remote --source "$GEM_SOURCE"` (which is what I was looking for when I found this issue/pr)